app/routes.py

```diff
@@ -1,2 +1,224 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import json, request, Blueprint, make_response, jsonify
```

Comment:
It doesn't look like you're using the `json` package here.

```suggestion
from flask import request, Blueprint, make_response, jsonify
```